dev-docs/bidders/sovrn.md

```diff
@@ -22,3 +23,160 @@ fpd_supported: true
+| `skip`           | optional | Indicates if the player will allow the video to be skipped, where 0 = no, 1 = yes                                              | `0`             | `integer`       |
+| `skipmin`        | optional | Only if the ad is skippable. Videos of total duration greater than this number of seconds can be skippable                     | `5`             | `integer`       |
+| `skipafter`      | optional | Only if the ad is skippable. Number of seconds a video must play before skipping is enabled                                    | `5`             | `integer`       |
+| `sequence`       | optional | For multiple ad in the same bid request. This value allow will for the coordinated delivery of multiple ad                     | `5`             | `integer`       |
+| `battr`          | optional | Blocked creative attributes                                                                                                    | `[1]`           | `integer array` |
+| `maxextended`    | optional | Max extended ad duration beyond the maxduration if extension is allowed. Blank or 0 - blocked. -1 - allowed without time limit | `5`             | `integer`       |
+| `minbitrate`     | optional | Minimum bit rate in Kbps                                                                                                       | `5`             | `integer`       |
+| `maxbitrate`     | optional | Maximum bit rate in Kbps                                                                                                       | `5`             | `integer`       |
+| `boxingallowed`  | optional | Indicates if letter-boxing of 4:3 content into a 16:9 window is allowed, where 0 = no, 1 = yes                                 | `5`             | `integer`       |
+| `playbackmethod` | optional | Playback methods that may be in use. See [Video Playback Methods](#video-playback-methods)                                     | `[1]`           | `integer array` |
+| `playbackend`    | optional | The event that causes playback to end. Refer to Playback Cessation Modes                                                       | `5`             | `integer`       |
+| `delivery`       | optional | Supported delivery methods (1 = streaming, 2 = progressive, 3 = download). If none specified, assume all are supported.        | `[1, 2]`        | `integer array` |
+| `pos`            | optional | Ad position on screen. Refer to [Ad Position](#ad-position)                                                                    | `5`             | `integer`       |
+| `api`            | optional | List of supported API frameworks for this impression. Refer to [API Frameworks](api-frameworks)                                | `[1, 2, 3]`     | `integer array` |
+ 
+
+### Note
+
+[Protocols list](https://docs.prebid.org/dev-docs/bidders/conversant.html#protocols)
```

Comment:
I find it confusing that you're referring to Conversant's protocols but copy all the rest of them here. But this has been open for so long, I'll merge this and let you decide whether you want to fix.